lib/event.ml

```diff
@@ -169,6 +169,32 @@ module Touch = struct
   (* Should return Dom.touchList *)
 end
 
+module Pointer = struct
+  include Mouse
+
+  include
+    [%js:
+    val pointerId : t -> int [@@js.get]
+
+    val pressure : t -> float [@@js.get]
+
+    val tangentialPressure : t -> float [@@js.get]
+
+    val tiltX : t -> float [@@js.get]
+
+    val tiltY : t -> float [@@js.get]
+
+    val twist : t -> float [@@js.get]
+
+    val width : t -> float [@@js.get]
+
+    val height : t -> float [@@js.get]
+
+    val pointerType : t -> string [@@js.get]
```

Comment:
I see this can be `'mouse' | 'pen' | 'touch'` (from mli). We will be able to leverage gen_js_api [great support for enum values](https://github.com/LexiFi/gen_js_api/blob/0321048c97ec645adfec18a5b966e0ae4080a043/TYPES.md#enums-mapped-to-polymorphic-variants-or-sum-types) in the future 😍